vendor/vendor.json

```diff
@@ -31,10 +31,10 @@
 			"revisionTime": "2016-08-04T10:47:26Z"
 		},
 		{
-			"checksumSHA1": "kxzfnN+IPF0sViI9iiUnI7aZVWM=",
+			"checksumSHA1": "ANMYkuezaW7Pl/JJBiEs0KLzTOg=",
 			"path": "github.com/go-sql-driver/mysql",
-			"revision": "d523deb1b23d913de5bdada721a6071e71283618",
-			"revisionTime": "2018-06-03T12:45:54Z",
+			"revision": "369b5d6e5e8e108ed4ae2f2b1607d444b3807dfb",
+			"revisionTime": "2018-11-13T02:38:49Z",
 			"version": "v1.4",
 			"versionExact": "v1.4.0"
```

Comment:
I think you need to run this with `govendor fetch github.com/go-sql-driver/mysql@v1.4`. This should correctly update all the vendor metadata.

Reply:
oh, you are right. seems like i updated to the latest master commit. i fixed it and pushed the changes.

thanks for pointing that out.